web/frontend/src/components/slam.vue

```diff
@@ -95,6 +104,77 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
   });
 };
 
+const executeMoveOnMap = async () => {
```

Comment:
From the 6 week check in with Eliot, we decided to pass in the position-only constraint as an extra param in this request

Reply:
Should we scope adding this to the `Constraints` message?

Reply:
What prevents 2 different `executeMoveOnMap` calls from happening concurrently, which would destroy the first `motionServiceReq`, making it uncancellable?

Reply:
window.localstorage usage (incoming)